runtime/src/main/java/org/corfudb/protocols/wireprotocol/orchestrator/ForceRemoveWorkflow.java

```diff
@@ -0,0 +1,69 @@
+package org.corfudb.protocols.wireprotocol.orchestrator;
+
+import lombok.Getter;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.view.Layout;
+
+import javax.annotation.Nonnull;
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+
+import static org.corfudb.protocols.wireprotocol.orchestrator.OrchestratorRequestType.FORCE_REMOVE;
+
+/**
+ * Created by Maithem on 12/13/17.
+ */
+public class ForceRemoveWorkflow implements IWorkflow {
+
+    final RemoveNodeRequest request;
+
+    @Getter
+    final UUID id;
+
+    public ForceRemoveWorkflow(Request request) {
+        this.id = UUID.randomUUID();
+        this.request = (RemoveNodeRequest) request;
+    }
+
+    @Override
+    public String getName() {
+        return FORCE_REMOVE.toString();
+    }
+
+    @Override
+    public List<Action> getActions() {
+        return Arrays.asList(new ForceRemoveNode());
+    }
+
+    class ForceRemoveNode extends Action {
+        @Override
+        public String getName() {
+            return "ForceRemoveNode";
+        }
+
+        @Override
+        public void impl(@Nonnull CorfuRuntime runtime) throws Exception {
+            changeStatus(ActionStatus.STARTED);
+
+            Layout currentLayout = (Layout) runtime.getLayoutView().getLayout().clone();
+            Layout newLayout = (Layout) currentLayout.clone();
+
+            newLayout.setEpoch(newLayout.getEpoch() + 1);
+
+            newLayout.getLayoutServers().remove(request.getEndpoint());
```

Comment:
Shouldn't this logic of removing different servers be encapsulated in a method that belongs to layout.